### ISSUE TITLE
Can't distinguish topmost window from stacked faded windows

### ISSUE THREAD

**Issue by LucasZamprogno** [ataraxie/Eyetrack#1]
If windows A B and C are all on top of each other and faded, then it's not clear which one is going to become foregrounded once you look over

**Comment by ataraxie** [comment]
I guess this is related to #6 with the overlapping environment, right?

**Comment by LucasZamprogno** [comment]
Related but not the same. That one is more about how the windows interact going in and out of view. Here I was talking about when a group of overlapping windows are all transparent, it's hard to even know which one will come back into view if I look over

**Comment by ataraxie** [comment]
Yes, got you. By related I just meant that it's both about issues with overlapping windows.
